cdap-app-fabric/src/main/java/io/cdap/cdap/support/handlers/SupportBundleHttpHandler.java

```diff
@@ -77,4 +81,26 @@ public void createSupportBundle(HttpRequest request, HttpResponder responder,
     String uuid = bundleService.generateSupportBundle(bundleConfig);
     responder.sendString(HttpResponseStatus.OK, uuid);
   }
+
+  /** ensure the user has visibility for the query request they made */
+  private void ensureVisibilityOnProgram(String namespace, String application, String programType, String programName,
```

Comment:
Given that we don't have the ability to individually perform permissions checks as the support bundle service concurrently generates bundles for the user, we should create a new `EntityId` for support bundles (for example, `SupportBundleId`) under the `InstanceId` entity and enforce on `StandardPermission` for those. That way, we can perform a single check per user on `StandardPermission.CREATE` for the support bundle instead of writing logic to check access for eah internal entity as that seems like unexpected behavior anyways.